Migrations.cs

```diff
@@ -28,7 +28,7 @@ public int Create()
                 .WithDescription("Customise common attributes on HTML element.")
                 .WithDisplayName("HTML Attributes"));
 
-            #endregion
+            #endregion HtmlAttributes
```

Comment:
What's prompted adding the region name to `#endregion` too?

Reply:
This is part of my CodeMaid set up, it doesn't effect anything and I think it makes everything easier to read

Reply:
Cool, was merely curiosity 👍